src/pyhf/tensor/tensorflow_backend.py

```diff
@@ -198,10 +204,10 @@ def abs(self, tensor):
         return tf.abs(tensor)
 
     def ones(self, shape):
-        return tf.ones(shape)
+        return tf.ones(shape, dtype=self.dtypemap['float'])
```

Comment:
aren't these ints by default?

Reply:
nope, they need to be floats (for use in the modifiers)